mkdocs/docs/HPC/jupyter.md

```diff
@@ -0,0 +1,129 @@
+
+## Using Jupyter Notebooks on the HPC
+
+### Starting a notebook using the web portal
+
+Through the web portal you can easily start a Jupyter notebook on a workernode, via the *Jupyter Notebook* button under the *Interactive Apps* menu item.
+
+<center>
+![image](img/ood_start_jupyter.png)
+</center>
+
+### Importing libraries
+
+Importing libraries in a notebook running on the HPC isn't as straight forward as on notebooks running on your local machine. You can only import libraries that are part of certain modules on the HPC. To find the right module, first look at the toolchain used by the Jupyter notebook version. This can be found when looking at the `JupyterNotebook version` field when creating a new Jupyter notebook session. In the image above `7.2.0` is the notebook version and `GCCcore 13.2.0`is the toolchain used.
+
+After checking the toolchain, you can search the correct module using a shell environment. you can do this by clicking on `Clusters`>`_login Shell Access` in the web portal.
+
+<center>
+![image](img/ood_jupyter_open_shell.png)
+</center>
+
+Using `module available` followed by a library name will print all modules in which your library is included. For example if you want to make use of SciPy, it will look like this:
+
+```shell 
+$ module available Scipy
+
+----------------------------------------------------------------------------------- /apps/gent/RHEL8/zen2-ib/modules/all ------------------------------------------------------------------------------------
+   SciPy-bundle/2019.10-foss-2019b-Python-2.7.16     SciPy-bundle/2020.03-intel-2020a-Python-3.8.2    SciPy-bundle/2021.05-gomkl-2021a                 SciPy-bundle/2022.05-intel-2022a
+   SciPy-bundle/2019.10-foss-2019b-Python-3.7.4      SciPy-bundle/2020.03-iomkl-2020a-Python-3.8.2    SciPy-bundle/2021.05-intel-2021a                 SciPy-bundle/2023.02-gfbf-2022b
+   SciPy-bundle/2019.10-intel-2019b-Python-2.7.16    SciPy-bundle/2020.11-foss-2020b-Python-2.7.18    SciPy-bundle/2021.10-foss-2021b-Python-2.7.18    SciPy-bundle/2023.07-gfbf-2023a
+   SciPy-bundle/2019.10-intel-2019b-Python-3.7.4     SciPy-bundle/2020.11-foss-2020b                  SciPy-bundle/2021.10-foss-2021b                  SciPy-bundle/2023.11-gfbf-2023b     (D)
+   SciPy-bundle/2020.03-foss-2020a-Python-3.8.2      SciPy-bundle/2020.11-intel-2020b                 SciPy-bundle/2021.10-intel-2021b                 scipy/1.4.1-foss-2019b-Python-3.7.4
+   SciPy-bundle/2020.03-intel-2020a-Python-2.7.18    SciPy-bundle/2021.05-foss-2021a                  SciPy-bundle/2022.05-foss-2022a
```

Comment:
Trim this down to a handful of modules, see for example what we have on cluster `shinx` currently